src/tools/annotation/BidirectionalTool.js

```diff
@@ -35,6 +35,7 @@ export default class BidirectionalTool extends BaseAnnotationTool {
         getMeasurementLocationCallback: emptyLocationCallback,
         textBox: '',
         shadow: '',
+        drawHandles: true,
```

Comment:
Does this conflict with `drawHandlesOnHover`?

Reply:
`// Draw the handles
      const handleOptions = {
        color,
        handleRadius,
        drawHandlesIfActive: drawHandlesOnHover,
      };

      // Draw the handles
      if (this.configuration.drawHandles) {
        drawHandles(context, eventData, data.handles, handleOptions);
      }`

Reply:
So the drawHandlesOnHover will be passed as option to drawHandles function, the drawHandle prop will take precedence over it.